openpype/hosts/flame/api/plugin.py

```diff
@@ -1,3 +1,618 @@
-# Creator plugin functions
+import re
+import os
+from Qt import QtWidgets, QtCore
+import openpype.api as openpype
+from openpype import style
+import openpype.hosts.flame as opflame
+from . import lib, pipeline
+from copy import deepcopy
+
+log = openpype.Logger().get_logger(__name__)
+
+
+class CreatorWidget(QtWidgets.QDialog):
+
+    # output items
+    items = dict()
+
+    def __init__(self, name, info, ui_inputs, parent=None):
+        super(CreatorWidget, self).__init__(parent)
+
+        self.setObjectName(name)
+
+        self.setWindowFlags(
+            QtCore.Qt.Window
+            | QtCore.Qt.CustomizeWindowHint
+            | QtCore.Qt.WindowTitleHint
+            | QtCore.Qt.WindowCloseButtonHint
+            | QtCore.Qt.WindowStaysOnTopHint
+        )
+        self.setWindowTitle(name or "Pype Creator Input")
+        self.resize(500, 700)
+
+        # Where inputs and labels are set
+        self.content_widget = [QtWidgets.QWidget(self)]
+        top_layout = QtWidgets.QFormLayout(self.content_widget[0])
+        top_layout.setObjectName("ContentLayout")
+        top_layout.addWidget(Spacer(5, self))
```

Comment:
```suggestion
        top_layout.addSpacing(5)
```

Reply:
thanks, implemented in gitkraken://repolink/33c731ef76967f4bd3f6ba8b287ecc0c52412f33/commit/389e90670ed5181d9dd45211950f67f9aec1cc5e?url=https%3A%2F%2Fgithub.com%2Fpypeclub%2FOpenPype.git